config-user-example.yml

```diff
@@ -1,9 +1,76 @@
 ###############################################################################
-# User's configuration file for the ESMValTool
+# Example user configuration file for ESMValTool
+###############################################################################
+#
+# Note for users:
+# --------------
+# Site-specific entries for different HPC centers are given at the bottom of
+# this file. Comment out/replace as needed.
+#
+###############################################################################
+#
+# Note for developers:
+# -------------------
+# Two identical copies of this file (``ESMValTool/config-user-example.yml`` and
+# ``ESMValCore/esmvalcore/config-user.yml``) exist. If you change one of it,
+# make sure to apply the changes to the other.
+#
 ###############################################################################
 ---
 
+# Directory where results are stored
+output_dir: ./esmvaltool_output
```

Comment:
```suggestion
output_dir: ~/esmvaltool_output
```

Reply:
probably better than some local dir in some greatly treed up dir, the user risks having `esmvaltool_output` dirs in a gazillion places

Reply:
Fully agree!